pkg/ui/src/views/cluster/containers/nodesOverview/index.tsx

```diff
@@ -133,8 +133,9 @@ class NodeList extends React.Component<LiveNodeListProps> {
         if (!!record.nodeId) {
           return (
             <React.Fragment>
-              <Link to={`/node/${record.nodeId}`}>
-                <Text>{record.nodeId}</Text>
+              <Link className="nodes-table__link" to={`/node/${record.nodeId}`}>
+                <Text>{`N`}</Text>
```

Comment:
Can you take another look at Anne's example? I think the `N` and the number are bolded together.

Reply:
Thank you, fixed.